js-variables-and-numbers_social-media-post/js/index.js

```diff
@@ -19,4 +25,10 @@ console.clear();
 
 // --v-- write your code here --v--
 
+console.log("Title:", title);
+console.log("Text:", text);
+console.log("Number of Likes:", numberOfLikes);
```

Comment:
I think you forgot to increase the number of likes by one and log the result into the console ;- )

Reply:
thanks! fixed that :)

Reply:
Looks good!